src/workflows/pdf/proposal/CustomProposalPdfFactory.ts

```diff
@@ -135,28 +136,12 @@ export class CustomProposalPdfFactory extends PdfFactory<
     /**
      * Emitters
      */
-    this.emit('render:proposal', data);
-
-    if (technicalReview) {
-      this.emit('render:technicalReview', technicalReview);
-    }
 
     if (attachments.length > 0) {
       this.meta.attachments = attachments;
       this.emit('fetch:attachmentsFileMeta', attachments);
     } else {
-      if (questionarySteps.length > 0) {
```

Comment:
I'm not too familiar with this code, but how come we no longer render the questionnaires, technicalReview or samples here?

Reply:
These data are already being passed to the proposal template, so these can be referenced in the template code(hbs) and used. There is no need for rendering the questionaries again, which becomes redundant. 

And moreover, these event emitters `render:questionnaires` and `render:samples` doesnt have a listeners. So removing this code doesn't do any harm to the existing flow. 

@simonfernandes Let me know if you need more clarification. 

Kindly approve this PR, if you are aligned with my explanation.

Reply:
Thanks for that